easybuild/framework/easyblock.py

```diff
@@ -1445,7 +1445,20 @@ def extensions_step(self, fetch=False):
 
     def package_step(self):
```

Comment:
@sntgluca: ok, maybe we can set up a conf call on this once we've had the chance to take a detailed look at this, or you can join one of the future EasyBuild conf calls (see https://github.com/hpcugent/easybuild/wiki/Conference-calls)

if you're not on the EB mailing list yet, do subscribe (https://lists.ugent.be/wws/info/easybuild)